snorkel/learning/gen_learning.py

```diff
@@ -134,12 +87,12 @@ def train(self, L, deps=(), LF_acc_priors=None, LF_acc_prior_default=0.7,
                      element is a tuple of the form 
                      (LF 1 index, LF 2 index, dependency type),
                      see snorkel.learning.constants
-        :param LF_acc_priors: An N-element list of prior probabilities for the
+        :param LF_acc_prior_weights: An N-element list of prior weights for the
             LF accuracies
-        :param LF_acc_prior_default: Default prior probability for each LF 
-            accuracy; if LF_acc_priors is unset, each LF will have this prior
+        :param LF_acc_prior_weight_default: Default prior for the weight of each LF
```

Comment:
Ah, actually none of these make clear it's in log scale. That's my bad. :)  Maybe just a note above in the doc string that all weights are in log scale?